src/services/web3/web3.plugin.ts

```diff
@@ -27,15 +28,18 @@ import walletlinkLogo from '@/assets/images/connectors/walletlink.svg';
 import i18n from '@/plugins/i18n';
 import { rpcProviderService } from '../rpc-provider/rpc-provider.service';
 import { web3Service } from './web3.service';
+import { TallyConnector } from './connectors/tally/tally.connector';
 
 export type Wallet =
   | 'metamask'
   | 'walletconnect'
   | 'gnosis'
   | 'walletlink'
-  | 'portis';
+  | 'portis'
+  | 'tally';
 export const SupportedWallets = [
   'metamask',
+  'tally',
```

Comment:
The only thing I'd like changed here is to move tally right under wallet connect

Reply:
Done